gnovm/tests/files/assign33.gno

```diff
@@ -0,0 +1,14 @@
+package main
+
+func foo() (int, bool) {
+	return 1, true
+}
+
+func main() {
+	var a, b int = foo()
+	println(a, b)
+}
+
+// Error:
+// main/files/assign33.gno:8:6: cannot use foo<VPBlock(3,0)>() (value of type bool) as int value in assignment
```

Comment:
```suggestion
// main/files/assign33.gno:8:6: cannot use foo<VPBlock(3,0)>() (value of type bool) as int value in assignment:
```

Please apply this to fix the failing test (see https://github.com/gnolang/gno/actions/runs/11701117443/job/32586531655?pr=3017)